tests/createPages.groovy

```diff
@@ -0,0 +1,71 @@
+import java.util.*
```

Comment:
Did you manage to use this file somewhere in the tests? or it's a file which can be used for manual tests?

Reply:
Yes, that's for manual testing only